docs/debug/README.md

```diff
@@ -1,3 +1,32 @@
+# How to DEBUG
+
+## DEBUG: start with process locally
+
+Start kruise-manager process locally, it can connect to a local or remote k8s-cluster.
+
+**Set up hostname**
+
+First, you should think of a fake hostname for your local machine, such as `power-machine`.
+
+Add this line into `/etc/hosts` file in which machine that runs kube-apiserver:
+```
+IP_OF_YOUR_LOCAL_MACHINE power-machine
+```
+
```

Comment:
Please add the following:

Note: "IP_OF_YOUR_LOCAL_MACHINE" cannot be loop device ip 127.0.0.1

Reply:
Do you mean if we execute `make run` in machine A and the minikube is deployed in machine B, then we should put `IP_OF_A  power-machine` into `/etc/hosts` of B, instead of `127.0.0.1 power-machine` ?

Reply:
I mean put
127.0.0.1 power-machine
in /etc/hosts does not work。we should put 
ETH_IP power-machine

Someone may think 127.0.0.1 is also the IP_OF_YOUR_LOCAL_MACHINE.

Reply:
done